Markdown/Markdown.sublime-syntax

```diff
@@ -608,6 +613,15 @@ contexts:
               scope: invalid.illegal.unexpected-whitespace.markdown
             - match: (?=\s*(?!\)))
               push: link-title
+  image-inline-attr:
+    - match: '([ ]*)(\{)' # spaces not allowed before the open paren, but we check for it anyway to mark it as invalid
```

Comment:
Since this is non-standard (i.e. not in commonmark), we should probably stay on the safe side and only highlight correct syntax.